### PR TITLE
Update bindings to v1.5-1

### PR DESCRIPTION
**Pull request by eloc3147**
This adds the following functions:

- `ftdi_setflowctrl_xonxoff`
- `ftdi_tciflush` (Replacement for deprecated `ftdi_usb_purge_rx_buffer`)
- `ftdi_tcoflush` (Replacement for deprecated `ftdi_usb_purge_tx_buffer`)
- `ftdi_tcioflush` (Replacement for deprecated `ftdi_usb_purge_buffers`)

This also adds the SIO_* values, as some of them are needed for `ftdi_setflowctrl_xonxoff`

### CODE REVIEW

**Comment by eloc3147** [comment]
Something bad happening with checking out the submodule in the tests, and not entirely sure why.
Anything I need to fix?

**Comment by tanriol** [comment]
Hello and thanks for the contribution!

These are unrelated CI problems, I'll deal with them in #15 and either pull this or ping you if there are any real problems!

**Comment by tanriol** [comment]
The CI problems have been fixed, please rebase.

File: src/pregenerated.rs [hunk 1]
```diff
@@ -46,7 +81,7 @@ impl ftdi_parity_type {
 }
 #[repr(transparent)]
 #[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
-pub struct ftdi_parity_type(pub u32);
```
Comment (tanriol) [inline src/pregenerated.rs:49]: These changes look a bit strange, not sure where they are coming from.

Comment (tanriol) [reply src/pregenerated.rs:49]: Looks like this is system-dependent, see rust-lang/rust-bindgen#1966.

Comment (eloc3147) [reply src/pregenerated.rs:49]: This is caused by a lack of a bindgen::Builder::default_macro_constant_type setting. I can submit another PR to fix.

Comment (tanriol) [reply src/pregenerated.rs:49]: Already fixed, sorry :-)

**Comment by tanriol** [comment]
Looks like there are no reported problems with alphas, so I've published v1.0.0.

**Pull request merged**